frontends/web/src/locales/en/app.json

```diff
@@ -402,6 +402,55 @@
+        "btcdirect": {
+          "disclaimer": {
+            "dataProtection": {
+              "link": "BTC Direct privacy policy",
+              "text": "The BitBoxApp does not collect personal data. BTC Direct collects necessary information as per their Privacy Policy to meet regulatory standards.",
+              "title": "Data Protection"
+            },
+            "kyc": {
+              "text": "Customer identity verification is required for all trades. Your account manager will guide you through the process and answer any questions you might have.",
+              "title": "Identity verification"
+            },
+            "partnership": {
+              "text": "In partnership with BTC Direct, a trusted European broker, BitBox offers large-scale over-the-counter (OTC) trading for Bitcoin, Ethereum, stablecoins and more with expert support directly in the BitBoxApp.",
+              "title": "Buy and sell with BTC Direct's Private Trading Desk"
+            },
+            "paymentMethods": {
+              "buy": "Buying bitcoin:",
+              "buy2": "Place large buy orders with your account manager, lock in a fixed price, and pay via bank transfer. Your crypto is delivered directly to your BitBox after payment.",
+              "fee": "Minimum trade amounts of €50,000 apply, with a transparent fee between 0.5% and 1%.",
+              "sell": "Sell bitcoin:",
+              "sell2": "Lock in a fixed OTC price when placing a sell order. EUR payment is made once you send crypto to BTC Direct. Personal assistance is available if needed.",
+              "title": "Payment Methods and Fees"
+            },
+            "personal": {
+              "text": "Each Private Trading client is assigned a dedicated account manager for personalized assistance via phone, video call, email, or chat. Buy and sell prices are immediately fixed when the order is placed, even if settlement occurs later. Orders are executed through BTC Direct's OTC desk, minimizing slippage.",
+              "title": "Personal service with fixed pricing"
+            },
+            "security": {
+              "link": "BitBox02 security threat model",
+              "text": "BTC Direct's Private Trading Desk is an external service not covered by the BitBox02 Security Threat model. However, BTC Direct is well-acquainted with BitBox products and offers secure, knowledgeable guidance. BitBox products verify crypto addresses during transactions to ensure secure delivery.",
+              "title": "Security"
+            },
+            "title": "Disclaimer"
+          },
+          "info": "Get a personal advisor, reduced fees and place orders directly through your contact at BTC Direct.",
+          "infobox": {
+            "introbtc": "Get personal assistance for high-value, over-the-counter (OTC) Bitcoin trades.",
+            "intromulti": "Get personal assistance for high-value over-the-counter (OTC) trades for Bitcoin, Ethereum stablecoins, and more.",
```

Comment:
nit: this should have been using i18n context https://www.i18next.com/translation-function/context

 i.e. `intro_btconly` and `intro_multi`

but I think it is ok to keep as is.